.msggen.json

```diff
@@ -195,6 +195,7 @@
             "CLOSINGD_SIGEXCHANGE": 4,
             "DUALOPEND_AWAITING_LOCKIN": 10,
             "DUALOPEND_OPEN_COMMITTED": 12,
+            "DUALOPEND_OPEN_COMMIT_READY": 13,
```

Comment:
We need to be careful with this: the enum numeric values are stored in the DB, so adding to the end is the right thing to do here (not to reorder states, causing faulty loads from DB), however IIRC there are places where we check for sets of states by using `<` and `>` on the numeric values. This would obviously fail.

Can we have a quick check through the code to see if there are numeric comparisons in the code on values for this enum?

Reply:
iiuc those have all been updated to be comprehensive swtich statements in a clean up Rusty did earlier this year, but I'll do a sweep to double check!